.envrc

```diff
@@ -57,6 +59,9 @@ info() {
     cat <<EOF
 ${bold}direnv: ${@}${reset}
 EOF
+    # info() calls happen multiple times in this script. The last info statement
+    # is the one that will be used to group Sentry issues
+    error_message=( "${@}" )
```

Comment:
These "info" lines:
<img width="917" alt="image" src="https://user-images.githubusercontent.com/44410/111000372-eefac900-834f-11eb-86fb-68ccfcbfaff2.png">

will be grouped as an issue through the last line:
<img width="465" alt="image" src="https://user-images.githubusercontent.com/44410/111000326-d7bbdb80-834f-11eb-974a-2da4dd4454ca.png">

Reply:
I have changed the code to be "info" instead of "warning".

Reply:
Should we update alerts so that it only triggers for "error" level?

Reply:
Thanks!

Bug noticed -> I can't make it work if using ">=" for the level:
![image](https://user-images.githubusercontent.com/44410/111501068-07396200-871b-11eb-98b0-b1900cffd0e7.png)

After few shenanigans with the alerting it now works:
<img width="1389" alt="image" src="https://user-images.githubusercontent.com/44410/111500551-811d1b80-871a-11eb-8ed2-bb19a618e536.png">